manticore/core/cpu/abstractcpu.py

```diff
@@ -462,22 +463,17 @@ def emulate(self, instruction):
         # line present.
         del emu
 
-    #Generic string representation
-    def __str__(self):
-        '''
-        Returns a string representation of cpu state
-        
-        :rtype: str
-        :return: name and current value for all the registers. 
-        '''
-        result = ""
+    def print_instruction(self):
```

Comment:
Since `print_instruction` and `print_registers` don't really print, can we rename them to something more descriptive, like `render_instruction`/`render_registers` or even `registers_str`?